### ISSUE TITLE
Is this project using tokio or async-std?

### ISSUE THREAD

**Issue by firasuke** [skonik/rust-async-downloader#1]
Why are both `tokio` and `async-std` included?

**Comment by skonik** [comment]
Hi there. Sorry for the delay.

There wasn't any strategy specific intention to use both of them. I just was learning rust and using some code examples related to downloading a file in an async way. At some point I needed to interact with files and found that `async-std` is a good choice fitting into async mode. I copied some code, pasted into my code and it just worked.

As I understand and explain for myself `tokio` is a runtime or a mechanism for making IO to work in async mode. `async-std` in my case just provides futures/tasks interface for running standard library operations in async way. `async-std` has its own runtime, but I don't use this one in the project. 
So, again, `tokio` -> provides runtime, `async-std` -> provide simple async API for std operations.
